int32 Illumination
int32 TurningLights
bool Reset
int32 DrivingMode
int32 MovCamera
int32 cameraExp
bool AutoPilot
